vars/multicloudDeploy.groovy

```diff
@@ -53,14 +57,35 @@ void call(parameters = [:]) {
             .withMandatoryProperty('source', null, { config.neoTargets })
 
         utils.pushToSWA([
-            step: STEP_NAME,
+            step         : STEP_NAME,
             stepParamKey1: 'enableZeroDowntimeDeployment',
-            stepParam1: config.enableZeroDowntimeDeployment
+            stepParam1   : config.enableZeroDowntimeDeployment
         ], config)
 
         def index = 1
         def deployments = [:]
 
+        if (parameters.cfCreateServices) {
+            def createServices = [:]
+            for (int i = 0; i < parameters.cfCreateServices.size(); i++) {
+                Map createServicesConfig = parameters.cfCreateServices[i]
+                createServices["Service Creation ${i + 1 > 1 ? i + 1 : ''}"] = {
```

Comment:
* why not `i>0` instead of `i+1>1`?

This would end up in (for e.g. 3 services
```
- Service Creation
- Service Creation 2
- Service Creation 3
```

What would be wrong with having `Service Creation 1` for the first entry? Wouldn't it be more reasonable to drop the index in case there is only one entry (parameters.cfCreateServices.size())``

By the way: when we run in parallel (or introduce shuffling again)  the order of the items in random. Would end up in e.g. 

```
- Service Creation 3
- Service Creation 
- Service Creation 2
```

Looks strange and the index is more the less not helpful during troubleshooting. Can we have somehow a better name here (e.g. outlines where a service is created?

Reply:
honest answer: copy and paste from what we had so far in the sdk pipeline without too much considerations for such details. I'm ok with discussing how to improve it, but I'd prefer to keep it simple if possible.

Reply:
[meta]

> but I'd prefer to keep it simple if possible.

I guess all of us are always anxious to keep things simple. You will not find anybody fighting for the opposite. The question is what is simple? This question is not always easy to answer. The strive for simplicity must be seen in the light of some condition which we should fulfil. E.g. each feature comes with some additional code and some additional complexity. Now it would be easy to get simple code by not having any feature implemented. The resulting code would be simple, but useless ... A feature could also be: "make the life of a troubleshooter easy by having understandable log output". The log output makes the code more complex and less simple, but there is a benefit from that ...